geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/process/tube/TubeSelectProcessTest.scala

```diff
@@ -31,8 +31,7 @@ class TubeSelectProcessTest extends Specification {
 
   sequential
 
-  val dtgField = org.locationtech.geomesa.accumulo.process.tube.DEFAULT_DTG_FIELD
```

Comment:
Humm... so seems like there are an implementation detail in having a field named "dtg" floating around. 

Previously, this static variable was marking the use and variable name.  I wonder if we should keep it?